source/_components/sensor.bme680.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: "BME680 Sensor"
+description: "Instructions how to integrate a BME680 sensor into Home Assistant."
+date: 2018-01-16 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: Sensor
+ha_release: 0.62
+ha_iot_class: "Local Push"
+---
+
+
+The `bme680` sensor platform allows you to read temperature, humidity, pressure and gas resistance values of a [Bosch BME680 Environmental sensor](https://cdn-shop.adafruit.com/product-files/3660/BME680.pdf) connected via [I2c](https://en.wikipedia.org/wiki/I²C) bus (SDA, SCL pins). It allows you to use all the operation modes of the sensor described in its datasheet.  In addition, it includes a basic air quality calculation that uses gas resistance and humidity measurements to calculate a percentage based air quality measurement.
+
+Tested devices:
+
+- [Raspberry Pi](https://www.raspberrypi.org/)
+
+To use your BME680 sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: bme680
+```
+
+Configuration variables:
+
+- **name** (*Optional*): The name of the sensor
+- **i2c_bus** (*Optional*): I2c bus where the sensor is. Defaults to 1, for Raspberry Pi 2, 3 Zero and Zero-W.
+- **i2c_address** (*Optional*): I2c address of the sensor. It is 0x76 or 0x77. Default 0x77.
+- **monitored_conditions** array  (*Optional*): Conditions to monitor. Available conditions are *temperature*, *humidity*, *pressure*, *gas* and *airquality*. By default *temperature*, *humidity*, *pressure* and *airquality* are enabled.
+- **oversampling_temperature** (*Optional*): Oversampling multiplier as described in the sensor datasheet. Can be 0 (no sampling), 1, 2, 4, 8, or 16. Default is 8.
+- **oversampling_pressure** (*Optional*): Oversampling multiplier as described in the sensor datasheet. Can be 0 (no sampling), 1, 2, 4, 8, or 16. Default is 2.
+- **oversampling_humidity** (*Optional*): Oversampling multiplier as described in the sensor datasheet. Can be 0 (no sampling), 1, 2, 4, 8, or 16. Default is 4.
+- **filter_size** (*Optional*): IIR filter size as described in the sensor datasheet. Can be 0 (off), 1, 3, 7, 15, 31, 63 or 127. Default is 3.
+- **gas_heater_temperature** (*Optional*): The temperature to heat the hotplate to for gas resistance measurements as descibed in the sensor datasheet.  Can be between 200-400&deg;C. Default is 320&deg;C.
+- **gas_heater_duration** (*Optional*): The duration to heat the hotplate in milliseconds for gas resistance measurements as descibed in the sensor datasheet.  Can be between 1-4032 ms. In reality you will likely need between 80-100ms to reach a stable temperature.  Using a duration greater than 1000ms is unadvisable as it will essentially result in the heater being continually on due to the 1 second update interval. Default is 150 ms.
```

Comment:
- `descibed` -> `described`
- Missing comma: `In reality you` -> `In reality, you`
- `unadvisable` -> `inadvisable`
- `1 second` -> `1-second`

Reply:
Fixed